viewer_server/client_db_access_handler.py

```diff
@@ -65,9 +68,9 @@ def construct_report_filter(report_filters):
             # severity value can be 0
             AND.append(Report.severity == report_filter.severity)
         if report_filter.suppressed:
-            AND.append(Report.suppressed == True)
+            AND.append(Report.suppressed == true())
```

Comment:
The reason for this is because of pep8 warning `E712: Comparison to False should be "if cond is False:" or "if not cond:"`
This can be fixed if we use [this style](http://stackoverflow.com/questions/18998010/flake8-complains-on-boolean-comparison-in-filter-clause)